pkg/integrations/prometheus/utils/operatorutils.go

```diff
@@ -260,3 +260,17 @@ func ExtractMatchingLabelSet(groupIDStr string, matchingLabels []string, include
 
 	return matchingLabelValStr
 }
+
+func CalculateStandardVariance(values []float64) float64 {
+	sum := 0.0
+	length := float64(len(values))
+	for _, val := range values {
+		sum += val
+	}
+	avg := sum / length
+	sumValSquare := 0.0
+	for _, val := range values {
+		sumValSquare += (val - avg) * (val - avg)
+	}
+	return sumValSquare / length
+}
```

Comment:
nit: Perhaps a better place for this function would be in the main `utils` package. Something like `/pkg/utils/mathutils.go`. What do you think? Then later if we wanted to for example find the variance of latencies for traces, we could get it from the main utils package instead of this prometheus utils package

Reply:
Yes, i agree